SasquatchMac/SasquatchMac/ViewControllers/AnalyticsViewController.swift

```diff
@@ -3,9 +3,24 @@ import Cocoa
 // FIXME: trackPage has been hidden in MSAnalytics temporarily. Use internal until the feature comes back.
 class AnalyticsViewController : NSViewController, NSTableViewDataSource, NSTableViewDelegate {
 
-  private enum CellIdentifiers {
-    static let keyCellId = "keyCellId"
-    static let valueCellId = "valueCellId"
+  class EventProperty : NSObject {
+    var key: String = ""
+    var type: String = EventPropertyType.String.rawValue
+    var string: String = ""
+    var double: NSNumber = 0
+    var long: NSNumber = 0
+    var boolean: Bool = false
+    var dateTime: Date = Date.init()
+  }
+
+  enum EventPropertyType : String {
+    case String = "String"
```

Comment:
Can you please make the enums camel cased? (not the string values though)
so `string`, `double`, `long`, `boolean`, and `dateTime`?

Reply:
Fixed this in new commit.